fms2_io/fms_io_utils.F90

```diff
@@ -25,7 +26,8 @@ module fms_io_utils_mod
 public :: get_data_type_string
 public :: get_checksum
 public :: open_check
-
+public :: string_compare
+public :: restart_filepath_mangle
```

Comment:
The `@internal` in the doxygen documentation for `restart_filepath_mangle` should also be removed.